docs/tempo/website/operations/architecture.md

```diff
@@ -24,13 +27,17 @@ the [Grafana Agent](https://github.com/grafana/agent) uses the otlp exporter/rec
 
 ### Ingester
 
-The Ingester batches trace into blocks, blooms, indexes, and flushes to the backend.  Blocks in the backend are generated in the following layout.
+The Ingester batches trace into blocks, creates bloom filters and indexes, and then flushes it all to the backend. 
+Blocks in the backend are generated in the following layout.
 
 ```
 <bucketname> / <tenantID> / <blockID> / <meta.json>
-.                                     / <bloom>
-.                                     / <index>
-.                                     / <data>
+                                      / <index>
+                                      / <data>
+                                      / <bloom_0>
+                                      / <bloom_1>
+                                        ...
+                                      / <bloom_n>
```

Comment:
We need another small change in the Querier section of this page. 

```
It begins by querying the ingesters to see if the id is currently stored there, if not it proceeds to use the bloom and indexes to find the trace in the storage backend.
```

This is no longer done sequentially, we can simplify this sentence.. "it searches the ingester and blocks for traces"